.github/workflows/cd.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Validate input
         run: |
-          [[ "${{ secrets.admin_token }}" ]] || { echo "Set ADMIN_TOKEN variable"; exit 1; }
+          [[ "${{ secrets.tb_admin_token }}" ]] || { echo "Set TB_ADMIN_TOKEN variable"; exit 1; }
```

Comment:
Can we rewrite this error message to be way more explicit? Suggestion:

`Go to tb_host/tokens, copy the "admin token (user@domain.com)" associated to a user account and set TB_ADMIN_TOKEN as a Secret in your Git repository`

expanding `tb_host`

Reply:
Finally changed to `Go the tokens section in your workspace and copy the "admin token (user@domain.com)" associated to a user account and set TB_ADMIN_TOKEN as a Secret in your Git repository` as any secret will be masked in the job output.